src/findex.rs

```diff
@@ -93,26 +93,28 @@ impl<
 {
     type Error = Error<Address<ADDRESS_LENGTH>>;
 
-    async fn search(&self, kw: &Keyword) -> Result<HashSet<Value>, Self::Error> {
-        let l = Self::hash_keyword(kw);
+    async fn search(&self, keyword: &Keyword) -> Result<HashSet<Value>, Self::Error> {
+        let l = Self::hash_keyword(keyword);
         let words = IVec::new(l, self.el.clone()).read().await?;
         (self.decode)(words).map_err(|e| Error::Conversion(format!("{e:?}")))
     }
 
     async fn insert(
         &self,
-        kw: Keyword,
-        vs: impl Sync + Send + IntoIterator<Item = Value>,
+        keyword: Keyword,
+        bindings: impl Sync + Send + IntoIterator<Item = Value>,
```

Comment:
No, `vs` = `values`

Reply:
right. Do you agree to also change it in `IndexADT` trait?

Reply:
okay for me